### PR TITLE
9083 api discharge verification

### PR DESCRIPTION
**Pull request by doug-lovett**
*Issue #:* /bcgov/entity#9083

*Description of changes:*
- Response data changes for non-financing registrations to support verification statement reports
- Report template updates to support discharge verification statements.
- Update unit tests.
- Update schema version to 1.4.5 to include recent general collateral, clientReferenceId definition changes. 
- Enable discharge payments as new code is now working in dev 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-api/src/ppr_api/resources/financing_statements.py [hunk 1]
```diff
@@ -559,25 +565,33 @@ def post(registration_num):
...
+                                        account_id)
             # Try to save the discharge statement: failure throws a business exception.
-            registration = Registration.create_from_json(request_json,
-                                                         model_utils.REG_CLASS_DISCHARGE,
-                                                         statement,
-                                                         registration_num,
-                                                         account_id)
-            registration.save()
-
+            # registration = Registration.create_from_json(request_json,
```
Comment (kialj876) [inline ppr-api/src/ppr_api/resources/financing_statements.py]: remove comment or are you planning to use this later?

File: ppr-api/src/ppr_api/resources/financing_statements.py [hunk 2]
```diff
@@ -613,12 +627,19 @@ def get(registration_num, discharge_registration_num):
                                                                  is_staff(jwt),
                                                                  registration_num)
 
+            response_json = statement.verification_json('dischargeRegistrationNumber')
             if resource_utils.is_pdf(request):
                 token = g.jwt_oidc_token_info
                 # Return report if request header Accept MIME type is application/pdf.
-                return get_pdf(statement.json, account_id, ReportTypes.DISCHARGE_STATEMENT_REPORT.value, token['name'])
+                # return get_pdf(statement.json, account_id, ReportTypes.DISCHARGE_STATEMENT_REPORT.value,
```
Comment (kialj876) [inline ppr-api/src/ppr_api/resources/financing_statements.py]: same thing here

**Review by kialj876 — APPROVED** [review]
Hey Doug, looks good just two sections of commented out code I think you can delete if you have time

File: ppr-api/src/ppr_api/resources/financing_statements.py (hunk 1 above)
Comment (doug-lovett) [reply ppr-api/src/ppr_api/resources/financing_statements.py]: Removed in latest commit..

File: ppr-api/src/ppr_api/resources/financing_statements.py (hunk 2 above)
Comment (doug-lovett) [reply ppr-api/src/ppr_api/resources/financing_statements.py]: Removed in latest commit.

**Review by jordiwes — APPROVED** [review]
LGTM

**Pull request merged**